backend/aws.go

```diff
@@ -78,6 +96,8 @@ type environment struct {
 	TotalVCPU        int     `json:"total_vcpu" groups:"summary,details"`
 	TotalMemoryGB    float32 `json:"total_memory_gb" groups:"summary,details"`
 	State            string  `json:"state" groups:"summary,details"`
+	BillsAccrued     string  `json:"bills_accrued" groups:"summary,details"`
+	BillsSaved       string  `json:"bills_saved" groups:"summary,details"`
```

Comment:
Would it be better to have a single number called `Cost` or something that can either be positive or negative depending on whether we paid or saved? @kgeorgiou  what do you think?

Reply:
Hmm. I feel it's necessary to have two stats both in functionality and in UI.
1. It wouldn't make sense to aggregate BillsSaved and BillsAccrued into one stat.
    - Add up BillsAccrued and BillsSaved and you get bills accrued w/o toggle. 
    - Subtract BillsSaved from BillsAccrued, and you get a delta between up and downtime
    In both cases, the purpose of the toggle wouldn't be conveyed clearly.

2. The Vue UI shows 2 columns of stats per env.